debian/rules

```diff
@@ -43,15 +43,15 @@ binary-syncd-rpc:
 #	-DCMAKE_LIBRARY_PATH=$(DEB_HOST_MULTIARCH)
 
 override_dh_auto_configure:
+    ifeq ($(shell cat /tmp/syncd-build), --enable-rpcserver=yes)
+	sed -i 's/ENABLE_SAITHRIFT=0/ENABLE_SAITHRIFT=1/' syncd/usr/bin/syncd_init_common.sh
+    else
+	sed -i 's/ENABLE_SAITHRIFT=1/ENABLE_SAITHRIFT=0/' syncd/usr/bin/syncd_init_common.sh
+    endif
```

Comment:
@lguohan: The behavior is as follows:

1. Try to find file in path relative to root (the level with the `debian/` directory.
2. If file is not found, try to find file relative to `debian/tmp/`